__tests__/app.test.js

```diff
@@ -79,3 +89,50 @@ describe("GET Requests", () => {
     });
   });
 });
+
+describe("PATCH Requests", () => {
+  describe("/api/articles/:article_id", () => {
+    test("status 200:updates given article and responds with the updated article (positive number)", () => {
+      const newObj = { inc_votes: 1 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(newObj)
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: 1,
+              body: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: 101,
+            })
+          );
+        });
+    });
+    test("status 200:updates given article and responds with the updated article (negative number)", () => {
+      const newObj = { inc_votes: -100 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(newObj)
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: 1,
+              body: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: 0,
```

Comment:
What happens if a negative number take the total votes below 0? What do you want to happen?